civis/io/_scripts.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+from civis import APIClient
+from civis.futures import CivisFuture
+
+log = logging.getLogger(__name__)
+
+
+def run_template(id, arguments, JSONValue=False):
+    """Run a template and return the results.
+
+    Parameters
+    ----------
+    id : int
+        The template id to be run.
+    arguments : dict
+        Dictionary of arguments to be passed to the template.
+    JSONValue : bool
+        If True, will return the JSON output of the template.
+        If False, will return the file ids associated with the
+        output results.
+
+    Returns
+    -------
+    If JSONValue = False:
+      file_ids : dict
+        Dictionary of file ids with the keys being their output names.
+    If JSONValue = True:
+      json_output : civis.response.Response
```

Comment:
does this return a `civis.response.Response` or a `dict`?  I think we would prefer the latter.

Reply:
as far as I can tell, `civis.response.Response` is a dict? At least it seems to behave as one. The reason I wrote `civis.response.Response` here is that's the class of the returned object from `client.scripts.list_containers_runs_outputs`

Reply:
Isn't there a call to subset that output to `value` so it's no longer a `civis.response.Response`?

Reply:
Oddly, `type(outputs[0].value)` is still `<class 'civis.response.Response'>`

Reply:
`Response` is a subclass of `dict` (see e.g. #228 ). I suggest casting the result as a `dict` before returning it.

Reply:
Interesting. 
Changed it to be a dict.